src/lib/logical_query_plan/lqp_utils.hpp

```diff
@@ -239,27 +240,30 @@ std::vector<std::shared_ptr<AbstractLQPNode>> lqp_find_leaves(const std::shared_
  *         This is a helper method that maps column ids from tables to the matching output expressions. Conceptually,
  *         it only works on data source nodes. Currently, these are StoredTableNodes, StaticTableNodes and MockNodes.
  */
-ExpressionUnorderedSet find_column_expressions(const AbstractLQPNode& lqp_node, const std::set<ColumnID>& column_ids);
+ExpressionUnorderedSet find_column_expressions(const AbstractLQPNode& lqp_node,
+                                               const std::vector<ColumnID>& column_ids);
 
 /**
- * @return True, if there is unique constraint in the given set of @param unique_constraints matching the given
- *         set of expressions. A unique constraint matches if it covers a subset of @param expressions.
+ * @return True, if there is unique column combination in the given set of @param unique_column_combinations matching
```

Comment:
"a unique column combination"?

Reply:
changed to "a UCC" to achieve a nicer formatting / brevity.